deepmerge/strategy/core.py

```diff
@@ -23,16 +41,18 @@ def _expand_strategy(cls, strategy):
 
         Otherwise, return the value, implicitly assuming it's a function.
         """
-        if isinstance(strategy, string_type):
-            method_name = "strategy_{0}".format(strategy)
-            if not hasattr(cls, method_name):
-                raise StrategyNotFound(strategy)
-            return getattr(cls, method_name)
+        if isinstance(strategy, str):
+            method_name = f"strategy_{strategy}"
+            if hasattr(cls, method_name):
+                return getattr(cls, method_name)
+            raise StrategyNotFound(strategy)
         return strategy
 
-    def __call__(self, *args, **kwargs):
+    def __call__(
```

Comment:
I intentionally didn't use the full types so it would be pretty easy to pass this through. I still kind of prefer that we don't have to have to duplicate the annotations here and in the strategy object, but I guess you kind of have to for typing reasons?

Reply:
>  I guess you kind of have to for typing reasons?
Yeah pretty much - without it it's ambiguous to both callers and the type checker for what these are. It is possible to use things like a param spec, but it's really not worth it for this use case.